src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.ts

```diff
@@ -180,13 +198,37 @@ export class ContentTypeFieldsAddRowComponent implements OnDestroy, OnInit {
     }
 
     private loadMessages(): void {
-        const i18nKeys = [...this.toolTips, 'contenttypes.dropzone.rows.add'];
+        const i18nKeys = [
+            ...this.toolTips,
+            'contenttypes.dropzone.rows.add',
+            'contenttypes.dropzone.rows.tab_divider'
+        ];
 
-        this.dotMessageService.getMessages(i18nKeys).subscribe((res) => {
+        this.dotMessageService.getMessages(i18nKeys).pipe(take(1)).subscribe((res) => {
             this.i18nMessages = res;
+            this.loadActions();
         });
     }
 
+    private loadActions(): void {
+        this.actions = [
+            {
+                label: this.i18nMessages['contenttypes.dropzone.rows.add'],
+                icon: 'fa fa-plus',
+                command: () => {
+                    this.setColumnSelect();
+                }
+            },
+            {
+                label: this.i18nMessages['contenttypes.dropzone.rows.tab_divider'],
+                icon: 'fa fa-plus',
+                command: () => {
+                    this.addTabDivider();
```

Comment:
If this is the only place where we use this method why don't just call the `this.dotEventsService.notify('add-tab-divider')` directly in the function?

Reply:
?